integrations/cloud-notifications/metadata.yaml

```diff
@@ -760,3 +760,41 @@
             - Notifications: The notifications which you want to be notified
           - **Integration configuration**
             - Alert Source API key: The key you copied in the ilert configuration step.
+
+- id: "notify-cloud-signl4"
```

Comment:
Please remove it from this file. It contains Netdata Cloud notifications. This is the Netdata Agent notification method (the metadata file correctly added in `src/health/notifications/signl4`).

Reply:
Ah, I see. Thanks and done.